activesupport/lib/active_support/cache/redis_cache_store.rb

```diff
@@ -0,0 +1,389 @@
+# frozen_string_literal: true
+
+begin
+  require "redis"
+  require "redis/distributed"
```

Comment:
Always require `redis/distributed`, even if we use a single Redis, so we can reference `Redis::Distributed::CannotDistribute` below.